### ISSUE TITLE
Provide pre-built release for installations that cannot use wpm

### ISSUE THREAD

**Issue by dlaflamme** [Wox-launcher/Wox.Plugin.Switcheroo#7]
Some environments only allow sanctioned browsers to make outgoing connections (such as firefox, chrome, etc.) and in such cases, "wpm install <plugin_name>" will not work. 

Could you provide a zip with a pre-built version of the Switcheroo plugin that can be unzipped into the Wox plugins folder?

**Comment by bao-qian** [comment]
you can set http proxy in wox or "internet options"

**Comment by dlaflamme** [comment]
Yes, that is one option that may work but isn't necessarily easy if a user doesn't know the proxy host/port or if the proxy is requires Kerberos authentication (which I haven't tested directly but I suspect does not work).

I was able to get around my proxy restrictions by doing the following (with a sanctioned browser that is configured to use the proxy):
1. Get http://api.getwox.com/plugin/search/switcheroo, extract plugin_file field
2. append the contents of plugin_file field to http://api.getwox.com/media/ and get that .wox file
3. Unzip .wox file into Wox plugins directory

This isn't necessarily easy either, but may be a useful option for those who can't use the built-in proxy option.

**Comment by bao-qian** [comment]
you can download the plugin file directly
http://www.getwox.com/plugin/52
there is a download button on the left panel.
then you could open is using wox or drag it to opening wox...
